lighthouse-core/report/html/templates.html

```diff
@@ -60,13 +60,37 @@
           <span class="lh-audit-group__title"></span>
           <span class="lh-audit-group__itemcount"></span>
           <!-- .lh-audit-group__description will be added here -->
+          <!-- .lh-metrics-toggle will be added here -->
         </div>
         <div class=""></div>
       </div>
     </summary>
   </details>
 </template>
 
+<!-- Lighthouse metrics toggle -->
+<template id="tmpl-lh-metrics-toggle">
+  <div class="lh-metrics-toggle">
```

Comment:
i'd like to use a `<input type=checkbox>` here (as the underlying control).  

we shouldn't need JS for this guy (except for ` this.metricAuditGroup.classList.toggle('lh-audit-group--metrics__show-descriptions')`)

this guide seems to be solid on implementation.. also including their resources at the bottom. https://webdesign.tutsplus.com/tutorials/how-to-make-custom-accessible-checkboxes-and-radio-buttons--cms-32074

Reply:
done, hows that?